lib/custom/src/MShop/Service/Provider/Payment/Payone.php

```diff
@@ -22,6 +22,138 @@ class Payone
 	implements \Aimeos\MShop\Service\Provider\Payment\Iface
 {
 	/**
+	 * Returns the data passed to the Omnipay library
+	 *
+	 * @param \Aimeos\MShop\Order\Item\Base\Iface $base Basket object
+	 * @param $orderid Unique order ID
+	 * @param array $params Request parameter if available
+	 */
+	protected function getData( \Aimeos\MShop\Order\Item\Base\Iface $base, $orderid, array $params )
+	{
+		$urls = $this->getPaymentUrls();
```

Comment:
Yes, that's right! I'll fix this in a few minutes / hours.

The method processOrder() method was overwritten because 

$parts = \Aimeos\MShop\Order\Manager\Base\Base::PARTS_SERVICE | \Aimeos\MShop\Order\Manager\Base\Base::PARTS_ADDRESS; 

had to be changed to 

$parts = \Aimeos\MShop\Order\Manager\Base\Base::PARTS_SERVICE | \Aimeos\MShop\Order\Manager\Base\Base::PARTS_ADDRESS | \Aimeos\MShop\Order\Manager\Base\Base::PARTS_PRODUCT; 

in order to access the information about products.